src/api/operations/clients.queries.graphql

```diff
@@ -34,6 +34,7 @@ query GetClientName($id: ID!) {
 
 query GetClientPermissions($id: ID!) {
   client(id: $id) {
+    id
     ...ClientPermissions
```

Comment:
`ClientPermissions` already has `id` on it, so I don't think this is doing anything